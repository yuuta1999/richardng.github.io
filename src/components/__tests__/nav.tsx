import React from 'react'
import { render, cleanup } from '@testing-library/react'

import Nav from '../nav'

afterEach(cleanup)

it('it works', () => {
  const { container } = render(<Nav />)

  expect(container.firstChild).toMatchSnapshot()
})
