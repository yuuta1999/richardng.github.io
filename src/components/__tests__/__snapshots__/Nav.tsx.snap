// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<ul
  class="sc-bdVaJa jAhDcB"
/>
`;
